pages/linux/btrfs.md

```diff
@@ -0,0 +1,23 @@
+# btrfs
+
+> A modern copy on write (CoW) filesystem for Linux.
```

Comment:
We tend to omit adjectives ("modern", "open source", "top quality") in the main description. Stick to describing what it is and what it does.

Reply:
I've just changed it. Can you review it again? Sorry for my messy commit. Still, haven't used with pr :sweat_smile:

Reply:
Hi, 
No problem, we can change the commit message once your PR is merged :)